ldms/src/contrib/sampler/gpu_metrics_sampler/gmg_ldms_util.c

```diff
@@ -81,7 +81,7 @@ const metric_t metricsDefinitions[] = {
         {.name = "perf_level", .type = LDMS_V_D64, .pf = (funcPtr_t) getPerfLevel},
         {.name = "power_usage (mW)", .type = LDMS_V_S32, .pf = (funcPtr_t) getPowerUsage},
```

Comment:
@VanshIntel it looks like almost all of these metric names have the unit string in them. What I'm asking is that you please:
-  remove the '(\<unit-name\>)' from _all_ of the metric names
- add the unit string you remove to an entry called .unit="<unit>". Don't include the '()' in that string

For example,

```
   . . .
  {.name = "power_usage", .unit="mW", .type = LDMS_V_S32, .pf = (funcPtr_t)getPowerUsage},
  {.name = "gpu_temp", .unit="Celsius", .type = LDMS_V_D64, .pf = (funcPtr_t) getGpuTemp},
   . . .
```

The other change I'm asking that you make is to change the commit message to something like:

"Modify the schema name and add the set metric unit names"

Reply:
@tom95858 Thanks for the pointing the issue, I've made the changes accordingly. Please let me know if there is any other changes require.

Reply:
Thanks @VanshIntel. Could you please change the commit message to something like:

```
Refactor schema and metric names.

The schema name is changed to "gpumetrics"

The metric names have been refactored to move the unit
string from the metric name to the .unit field of the schema
metric data structure.
```

Reply:
@VanshIntel also please squash all of this into a single commit.

Reply:
@tom95858 I've merge all the changes into single commit. Please let me know if it require any changes.

Reply:
@tom95858 Sorry for the confusion. I've just updated the comment. Please let me know if there is any changes require. Thanks

Reply:
@tom95858 I've pushed the changes. Please let me know if you require any changes.

Reply:
@tom95858 Can you please approve this merge or Please let me know if any changes require.